LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -60,14 +59,15 @@ public class DefaultLoRaDataRequestHandler : ILoRaDataRequestHandler
             this.receiveWindowHits = meter?.CreateCounter<int>(MetricRegistry.ReceiveWindowHits);
             this.d2cPayloadSizeHistogram = meter?.CreateHistogram<int>(MetricRegistry.D2CMessageSize);
             this.c2dMessageTooLong = meter?.CreateCounter<int>(MetricRegistry.C2DMessageTooLong);
-            this.unhandledExceptionCount = meter?.CreateCounter<int>(MetricRegistry.UnhandledExceptions);
         }
 
         public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaRequest request, LoRaDevice loRaDevice)
         {
             if (request is null) throw new ArgumentNullException(nameof(request));
             if (loRaDevice is null) throw new ArgumentNullException(nameof(loRaDevice));
 
+            var deferredTasks = new List<Task>();
```

Comment:
If we do not have a downstream message or any other of the edge cases triggering the unobserved tasks, this allocation will never be used. I would suggest to wrap it in a local function and only allocate when we need it.

Reply:
Makes sense, applied here: af9adfef245ee9a06e6851b3b8a610d4209e35c7